### PR TITLE
[SE-0296] "await try" -> "try await" based on review feedback

### PR DESCRIPTION
**Pull request by DougGregor**

### CODE REVIEW

File: proposals/0296-async-await.md [hunk 1]
```diff
@@ -684,10 +684,12 @@ None of the concerns for top-level code affect the fundamental mechanisms of asy
 
 ## Revision history
 
+* Post-review changes:
+   * Replaced `try await` with `try await`.
```
Comment (PsychoH13) [inline proposals/0296-async-await.md:688]: I think you meant `   * Replaced `await try` with `try await`.`

File: proposals/0296-async-await.md [hunk 2]
```diff
@@ -682,12 +683,35 @@ top-level variables.
 
 None of the concerns for top-level code affect the fundamental mechanisms of async/await as defined in this proposal.
 
+### Await as syntactic sugar
+
+This proposal makes `async` functions a core part of the Swift type system, distinct from synchronos functions. An alternative design would leave the type system unchanged, and instead make `async` and `await` syntactic sugar over some `Future<T, Error>` type, e.g.,
```
[the remark this answers is not in the document]
Comment (docksr) [reply proposals/0296-async-await.md:688]: you should replace the"" await try'" with the"" try await [more"".](https://lotterysambadresult.su)

**Comment by docksr** [comment]
> Nit: if there `wa`

change this qoute plz[https://lotterysambadresult.su](url)

**Comment by docksr** [comment]
undo?( post review changes, instead of second [pitch)https://lotterysambadresult.su

**Pull request merged**